module/VuFind/src/VuFind/Db/Row/User.php

```diff
@@ -232,21 +232,38 @@ protected function encryptOrDecrypt($text, $encrypt = true)
             return $text;
         }
 
+        $configAuth = $this->config->Authentication;
+
         // Load encryption key from configuration if not already present:
-        if (null === $this->encryptionKey) {
-            if (!isset($this->config->Authentication->ils_encryption_key)
-                || empty($this->config->Authentication->ils_encryption_key)
+        if ($this->encryptionKey === null) {
+            if (!isset($configAuth->ils_encryption_key)
```

Comment:
`empty` includes a built in `isset` check and does not trigger notices for missing values, so this code can be simplified to only make the empty check.